csharp/ql/test/library-tests/dataflow/library/FlowSummaries.expected

```diff
@@ -1217,10 +1217,15 @@
 | System.IO.UnmanagedMemoryStreamWrapper.ToArray() | argument -1 -> return (normal) | false |
 | System.IO.UnmanagedMemoryStreamWrapper.Write(Byte[], int, int) | argument 0 -> argument -1 | false |
 | System.IO.UnmanagedMemoryStreamWrapper.WriteAsync(Byte[], int, int, CancellationToken) | argument 0 -> argument -1 | false |
+| System.Int32.Parse(ReadOnlySpan<Char>, NumberStyles, IFormatProvider) | argument 0 -> return (normal) | false |
 | System.Int32.Parse(string) | argument 0 -> return (normal) | false |
 | System.Int32.Parse(string, IFormatProvider) | argument 0 -> return (normal) | false |
 | System.Int32.Parse(string, NumberStyles) | argument 0 -> return (normal) | false |
 | System.Int32.Parse(string, NumberStyles, IFormatProvider) | argument 0 -> return (normal) | false |
+| System.Int32.TryParse(ReadOnlySpan<Char>, NumberStyles, IFormatProvider, out int) | argument 0 -> argument 3 | false |
+| System.Int32.TryParse(ReadOnlySpan<Char>, NumberStyles, IFormatProvider, out int) | argument 0 -> return (normal) | false |
+| System.Int32.TryParse(ReadOnlySpan<Char>, out int) | argument 0 -> argument 1 | false |
+| System.Int32.TryParse(ReadOnlySpan<Char>, out int) | argument 0 -> return (normal) | false |
```

Comment:
`SystemInt32Struct::getTryParseMethod()` only returns the `TryParse(string,...` methods. Should we cover only those? What do you think @hvitved?

Reply:
I think it would be good to cover these as well. However, the input specification should say `Element of`, e.g.
```
"System;Int32;false;TryParse;(System.ReadOnlySpan<System.Char>,System.Int32);;Element of Argument[0];Argument[1];taint"
```